packages/integrations/vercel/README.md

```diff
@@ -262,6 +262,82 @@ You can use Vercel middleware to intercept a request and redirect before sending
 > **Warning**
 > **Trying to rewrite?** Currently rewriting a request with middleware only works for static files.
 
+### Vercel Edge Middleware with Astro middleware
+
+The `@astrojs/vercel/serverless` adapter can automatically create the vercel edge middleware when you have an Astro middleware in you code base.
+
+This is an opt-in feature, and the `build.excludeMiddleware` option needs to be set to `true`:
+
+```js
+// astro.config.mjs
+import {defineConfig} from "astro/config";
+import vercel from "@astrojs/vercel";
+export default defineConfig({
+   output: "server",
+   adapter: vercel(),
+   build: {
+       excludeMiddleware: true
+   }
+})
+```
+
+Optionally, you can create a known file called `vercel-edge-middleware.(js|ts)` in the [`srcDir`](https://docs.astro.build/en/reference/configuration-reference/#srcdir) folder to create [`Astro.locals`](https://docs.astro.build/en/reference/api-reference/#astrolocals).
```

Comment:
```suggestion
Optionally, you can create a file recognized by Vercel named `vercel-edge-middleware.(js|ts)` in the [`srcDir`](https://docs.astro.build/en/reference/configuration-reference/#srcdir) folder to create [`Astro.locals`](https://docs.astro.build/en/reference/api-reference/#astrolocals).
```
suggest replacing "known" with "recognized by..." whatever understands/recognizes this file!

Reply:
It's not recognized by Vercel, but it's recognized by the adapter (us) :)